doc/classes/NativeMenu.xml

```diff
@@ -78,7 +81,8 @@
 				An [param accelerator] can optionally be defined, which is a keyboard shortcut that can be pressed to trigger the menu button even if it's not currently open. The [param accelerator] is generally a combination of [enum KeyModifierMask]s and [enum Key]s using bitwise OR such as [code]KEY_MASK_CTRL | KEY_A[/code] ([kbd]Ctrl + A[/kbd]).
 				[b]Note:[/b] Radio-checkable items just display a checkmark, but don't have any built-in checking behavior and must be checked/unchecked manually. See [method set_item_checked] for more info on how to control it.
 				[b]Note:[/b] The [param callback] and [param key_callback] Callables need to accept exactly one Variant parameter, the parameter passed to the Callables will be the value passed to [param tag].
-				[b]Note:[/b] This method is implemented only on macOS.
+				[b]Note:[/b] This method is implemented on macOS and Windows.
+				[b]Note:[/b] [param accelerator] and [param key_callback] are ignored on Windows.
```

Comment:
VERY nitpicky: I'd personally revert all of these, so that anyone not interested can set their hearts at peace as soon as they read the first 2 words.
```suggestion
				[b]Note:[/b] On Windows, [param accelerator] and [param key_callback] are ignored.
```